### PR TITLE
Fix markdown in WordArray lint

### PR DESCRIPTION
**Pull request by mrcasals**
Fixes the Markdown style for the `WordArray` lint.

### CODE REVIEW

**Comment by pocke** [comment]
Thank you for your pull-request!

The test is failing. Could you execute `rake generate_cops_documentation` and commit and squash the change that is generated by the task?

File: lib/rubocop/cop/style/word_array.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ module Style
       #
       # Configuration option: MinSize
       # If set, arrays with fewer elements than this value will not trigger the
-      # cop. For example, a `MinSize of `3` will not enforce a style on an array
+      # cop. For example, a `MinSize` of `3` will not enforce a style on an array
```
Comment (pocke) [inline lib/rubocop/cop/style/word_array.rb]: Sorry, there is another cause of CI failure.
This line is too long.

https://ci.appveyor.com/project/bbatsov/rubocop/build/952/job/qfvngr1ko9069oid

```
Offenses:
lib/rubocop/cop/style/word_array.rb:14:81: C: Line is too long. [81/80]
      # cop. For example, a `MinSize` of `3` will not enforce a style on an array
                                                                                ^
1009 files inspected, 1 offense detected

```

Comment (mrcasals) [reply lib/rubocop/cop/style/word_array.rb]: Thanks! I moved "array" to the next line and regenerated the docs with the `rake` task, let's hope this is fixed now 😄

**Comment by bbatsov** [comment]
❤️ 🙇

**Pull request merged**